docker/Dockerfile

```diff
@@ -73,3 +73,7 @@ RUN pip install --no-cache-dir multiscale_spatial_image jupyterlab-git \
         datalad-fuse
 
 RUN pip install --no-cache-dir brainrender napari
+
+# Neovim editor
+RUN apt-get install neovim
+RUN apt-get install python3-neovim
```

Comment:
- neovim should be added in the apt-section near the top that uses privileged user
- python part can be installed from conda-forge

also you may want to check if neovim itself can be installed from conda forge.

Reply:
also equivalent setting should be applied to Dockerfile.gpu and matlab otherwise the environments will not be the same.

Reply:
Moved the neovim insaller to under USER $NB_USER . Removed the python (to be added later by any user using conda or pip). Did the same for Docker.gpu and Docker.matlab files.